assets/docker-compose.yml

```
@@ -170,3 +170,4 @@ services:
           - ${SERVICE_NETWORK_NAME}-kong.${SERVICE_NETWORK_NAME}
     volumes:
       - ${PONGO_WD}:/kong-plugin
+      - ${PONGO_WD}/servroot/logs:/kong-prefix
```

Isn't this missing the "logs" part?
```suggestion
      - ${PONGO_WD}/servroot/logs:/kong-prefix/logs
```

---

will this work if `${PONGO_WD}/servroot/logs` doesn't exist?

---

> will this work if ${PONGO_WD}/servroot/logs doesn't exist?
yes, docker by default create missing paths as directory, which is what we want

---

> Isn't this missing the "logs" part?

you are right, applied